### PR TITLE
Fix case response

### PR DESCRIPTION
**Pull request by jaumef**
From #4 
Requires [update poweremail](https://github.com/gisce/poweremail/pull/28)

Fix errors on case response:
- Missing field "pem_mail_orig" on case reply new mail creation
- Removed call "complete_mail()" before mail is sent. Fails because server does not have the mail yet and it tries to retrieve it.
- Added __history to case reply

### CODE REVIEW

File: crm.py [hunk 1]
```diff
@@ -68,6 +70,7 @@ def email_send(self, cursor, uid, case, emails, body, context=None):
         pm_mailbox_obj.create(cursor, uid, {
             'pem_from': emailfrom,
             'pem_to': ', '.join(set(emails)),
+            'pem_mail_orig': case.conversation_id.mails[-1].pem_mail_orig or '',
```
Comment (ecarreras) [inline crm.py]: You're putting the other content email to the new email?

File: crm.py [hunk 2]
```diff
@@ -154,4 +157,29 @@ def _conversation_mails(self, cursor, uid, ids, field_name, args,
...
+                obj = self.pool.get('crm.case.history')
+                data['description'] = case.description
+                data['email'] = email or \
+                                user_obj.browse(cr, uid,
+                                                uid).address_id.email or False
+            obj.create(cr, uid, data, context)
+        return True
+
+    _history = __history
+
```
Comment (ecarreras) [inline crm.py]: What is needed this? I think we can use a fix in the core https://github.com/gisce/erp/pull/3929/files#diff-f14d4d014cf149259e03647f2450fb76

File: crm.py (hunk 1 above)
Comment (jaumef) [inline crm.py]: For what I understand, poweremail needs the last mail of the conversation in pem_mail_orig.

- This field is an email in plain-text format (unicode string)
- This field is only used to override the subject

File: crm.py (hunk 2 above)
Comment (jaumef) [inline crm.py]: Yup, bad method call made me think it wasn't being inherited. Solving this on next commits

File: crm.py (hunk 1 above)
Comment (ecarreras) [inline crm.py]: If you are in a case, you already have a `conversation_id` which can be assigned in the `create` values. https://github.com/gisce/crm_poweremail/blob/master/crm.py#L144

Comment (jaumef) [inline crm.py]: Conversation_id is set correctly.
The problem is the email creation (https://github.com/gisce/poweremail/blob/v5_backport/poweremail_mailbox.py#L284). 
If no pem_mail_orig is provided, this line fails. Maybe we should update poweremail instead?

Comment (ecarreras) [inline crm.py]: It must be fixed there, `create` only must use `pem_mail_orig` if this exists

**Pull request merged**